requirements/test-mindeps.txt

```diff
@@ -13,9 +13,9 @@ cffi==1.15.1
 chardet==3.0.4
     # via requests
 coverage==7.2.1
-    # via -r test.in
-cryptography==3.3.1
-    # via -r test-mindeps.in
+    # via -r requirements/test.in
```

Comment:
Hmm...Dependabot is going to fight us about the `$CWD`. We probably need to re-root this in the script.